hack/terraform/aws/variables.tf

```diff
@@ -78,11 +78,11 @@ variable "aws_vm_os_types_agented" {
       ami_owner       = "099720109477"
     },
     {
-      name            = "ubuntu3"
-      login           = "ubuntu"
-      init            = "init_script_ubuntu.sh"
-      ami_name_search = "ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*"
-      ami_owner       = "099720109477"
+      name            = "rhel"
```

Comment:
probably rename to rhel-host3, and previous ones to ubuntu-host1, and host2

Reply:
Sure. I missed that.

Reply:
Done